ansible/roles/stack-sunbird/tasks/inbound.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
We are no more supporting docker deployments. So this file is not required.

Reply:
Removed file.